html/elements/audio.json

```diff
@@ -57,36 +57,40 @@
             ],
             "support": {
               "chrome": {
-                "version_added": "3"
+                "version_added": "3",
+                "version_removed": "67"
               },
               "chrome_android": "mirror",
               "edge": {
-                "version_added": "12"
+                "version_added": "12",
+                "version_removed": "79"
               },
               "firefox": {
-                "version_added": "3.5"
+                "version_added": "3.5",
+                "version_removed": "67"
               },
               "firefox_android": "mirror",
               "ie": {
                 "version_added": "9"
               },
               "oculus": "mirror",
               "opera": {
-                "version_added": "10.5"
+                "version_added": "10.5",
+                "version_removed": "53"
               },
               "opera_android": {
-                "version_added": "11"
+                "version_added": "11",
+                "version_removed": "53"
               },
               "safari": {
-                "version_added": "3.1"
+                "version_added": "3.1",
+                "version_removed": "17"
               },
               "safari_ios": {
-                "version_added": "3"
+                "version_added": false
```

Comment:
Are you sure Safari iOS never supported this?

Reply:
I am -- I verified this via manual testing and confirmed that audio never autoplayed on iOS!